xml/cl.xml

```diff
@@ -488,7 +488,12 @@ server's OpenCL/api-docs repository.
     <enums start="-1102" end="-1105" name="ErrorCodes.1102" vendor="Qualcomm" comment="Reserved for Qualcomm extensions">
     </enums>
 
-    <enums start="-1106" end="-9999" name="ErrorCodes.1106" vendor="Khronos" comment="RESERVED FOR FUTURE ALLOCATIONS BY KHRONOS">
+    <enums start="-1106" end="-1107" name="ErrorCodes.1106" vendor="Intel" comment="Intel FPGA extension">
+        <enum value="-1106"         name="CL_PIPE_FULL_INTEL"/>
+        <enum value="-1107"         name="CL_PIPE_EMPTY_INTEL"/>
+    </enums>
+
+    <enums start="-1108" end="-9999" name="ErrorCodes.1106" vendor="Khronos" comment="RESERVED FOR FUTURE ALLOCATIONS BY KHRONOS">
```

Comment:
I'm not aware of any actual requirement to keep names unique, but in keeping with the rest of the file this would be `name="ErrorCodes.1108"`.

Reply:
Thanks.  Fixed.